src/assets/TokenListController.test.ts

```diff
@@ -21,9 +20,7 @@ const timestamp = Date.now();
 
 const staticTokenList: TokenListMap = {};
 for (const tokenAddress in contractMap) {
-  const { erc20, logo: filePath, ...token } = (contractMap as ContractMap)[
-    tokenAddress
-  ];
+  const { erc20, logo: filePath, ...token } = contractMap[tokenAddress];
```

Comment:
Types were added in this PR for `@metamask/contract-metadata`, so typecasting is no longer necessary.